kernel/kernel-api/src/test/java/io/delta/kernel/internal/types/JsonHandlerTestImpl.java

```diff
@@ -160,7 +157,17 @@ private static Object decodeElement(JsonNode jsonValue, DataType dataType) {
                     final Object parsedElement = decodeElement(element, arrayType.getElementType());
                     output.add(parsedElement);
                 }
-                return output;
+                return new ArrayValue() {
```

Comment:
If this becomes a common pattern, add a utility method to convert `List` to `ArrayValue`.